vic/drivers/classic/src/read_soilparam.c

```diff
@@ -732,15 +732,15 @@ read_soilparam(FILE *soilparam,
             temp.cell_area = dist; /* Grid cell area in m^2. */
         }
 
-        /*************************************************
-           Allocate and Initialize Snow Band Parameters
-        *************************************************/
-        Nbands = options.SNOW_BAND;
-        temp.AreaFract = (double *) calloc(Nbands, sizeof(double));
-        temp.BandElev = (double *) calloc(Nbands, sizeof(double));
-        temp.Tfactor = (double *) calloc(Nbands, sizeof(double));
-        temp.Pfactor = (double *) calloc(Nbands, sizeof(double));
-        temp.AboveTreeLine = (bool *) calloc(Nbands, sizeof(bool));
+            /*************************************************
+               Allocate and Initialize Snow Band Parameters
+            *************************************************/
+            Nbands = options.SNOW_BAND;
+            temp.AreaFract = calloc(Nbands, sizeof(*temp.AreaFract));
```

Comment:
The reason I think it is fine here is that there is no confusion, although it would be fine if you did change it.